packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -504,7 +504,8 @@ export default class TabAdvanced extends Component {
                     <AffinityHost vm={vm} />
                   </td>
                 </tr>
-                {vm.virtualizationMode === 'hvm' && (
+                {(vm.virtualizationMode === 'hvm' ||
+                  vm.virtualizationMode === 'pvhvm') && (
```

Comment:
I'm not sure this is the correct fix, AFAIU, PVHVM should be treated both as PV and HVM, maybe we should fix our objects instead with `isPv` and `ivHvm` properties?

/cc @olivierlambert @pdonias

Reply:
PVHVM is not "both" PV and HVM, it's a HVM with PV drivers (HVM first)

Reply:
So, the changes are good? Do you want me to change something?